.github/CONTRIBUTING.md

```diff
@@ -150,18 +159,73 @@ about how it works.
 
 ### Documentation
 
-The main documentation is written in [Markdown][] using [MkDocs][]. Dredd uses
-[ReadTheDocs][] to build and publish the documentation:
+Dredd's documentation is written in [Markdown][] using [Sphinx][]. [ReadTheDocs][] is used to build and publish the documentation:
 
 - [https://dredd.readthedocs.io](https://dredd.readthedocs.io) - preferred long URL
 - [http://dredd.rtfd.org](http://dredd.rtfd.org) - preferred short URL
 
-Source of the documentation can be found in the [docs][] directory. To contribute to Dredd's documentation, you will need to follow the [MkDocs installation instructions](http://www.mkdocs.org/#installation). Once installed, you may use following commands:
+Source of the documentation can be found in the [docs][] directory. To render Dredd's documentation on your computer, you need Python 3 and Node.js installed.
```

Comment:
Yeah, the reason I asked is that lots of libraries (even those that support Py 2.7) dropped Python 3 < 3.3 support because some 2.7 behaviours were reintroduced (`%` etc).

Looking at Sphinx release notes, they did drop Python 3.3 too (https://www.sphinx-doc.org/en/stable/changes.html#release-1-5-released-dec-5-2016)

> Drop python 2.6 and 3.3 support